renderer/containers/preferences.js

```diff
@@ -155,7 +155,11 @@ export default class PreferencesContainer extends Container {
 
     if (!newValue || await this.systemPermissions.ensureMicrophonePermissions()) {
       if (newValue) {
-        await this.getAudioDevices();
+        try {
```

Comment:
Isn't this moot as `getAudioDevices` cannot fail? Since we already wrap it in try/catch at https://github.com/wulkano/kap/pull/828/files#diff-c8ec885dc71d0b6d7444b52b062ebfa2R214

Reply:
This is calling `getAudioDevices` of this container class, which calls the main one and then does some computations. I added that as an additional fail check to make sure whatever goes in that method doesn't stop the toggle for being enabled

Reply:
Okey. I think this one too should call `showError(error, {reportToSentry: true});` instead of just logging, so the user can better report to us any possible issues.

Reply:
Done 👍 Might be worth looking into using electron-unhandled to make sure we capture everything. Should help with people reporting as well